docs/docs/create-source-plugin.md

```diff
@@ -74,7 +74,7 @@ exports.sourceNodes = async ({ boundActionCreators }) => {
 };
 ```
 
-Puruse the [`sourceNodes`](/docs/node-apis/#sourceNodes) and
+Peruse the [`sourceNodes`](/docs/node-apis/#sourceNodes) and
```

Comment:
You tried to fix a typo & made new one. It must be `Pursue` 😂

Reply:
@deadcoder0904 I think `Peruse` is valid :)